### DISCUSSION TITLE
Mail settings

### DISCUSSION THREAD

**Discussion by Solverz-0** [firefly-iii/firefly-iii#6513] [Firefly III] [answered]
Does all of the below relate the emails that Firefly can send out or notifications in the firefly gui or something else?

> Firefly III can send you the following messages.
> SEND_REGISTRATION_MAIL=true
> SEND_ERROR_MESSAGE=true
> SEND_LOGIN_NEW_IP_WARNING=true
> 
> These messages contain (sensitive) transaction information:
> SEND_REPORT_JOURNALS=true

**Comment by JC5** [comment, marked as answer]
Yes. It will change in the future but for now, yes.

**Reply by Solverz-0** [reply]
Sorry was that yes to emails or notifications?

**Reply by JC5** [reply]
All these settings are about emails.
